src/components/Avatar/AvatarDropdownContainer.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+const propTypes = {
+    tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+    className: PropTypes.string,
+    wrapperClassName: PropTypes.string,
+   
+}
+
+const defaultProps = {
+    tag: 'div',
+}
+
+const AvatarDropdownContainer = (props) => {
+    const { className, tag:Tag, wrapperClassName, ...attributes } = props
+    const wrapperClasses = classNames('avatar avatar-dropdown size-sm', wrapperClassName)
+    return (
+        <Tag className={wrapperClasses}>
+            <ul {...attributes} className={listClasses}></ul>
+        </Tag>
+    )
+}
```

Comment:
This looks to me an `AvatarIcon` component with a `ul` content.

Reply:
Sorry ! This Component isn't actually used anywhere! I made it and before implementing the Overlapping Avatars and forgot to delete it.